pkg/account/session/config.go

```diff
@@ -0,0 +1,22 @@
+// Copyright © 2022 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package session
+
+import "time"
+
+// SessionConfig is the configuration for user sessions.
+type Config struct {
+	Duration time.Duration `name:"duration" description:"Time after which user sessions will automatically expire (default 8760h)"`
```

Comment:
You don't need to put the default in the description. It is generated automatically as long as it was set in the `cmd` package.